src/nextjs/nextjs-wizard.ts

```diff
@@ -701,3 +715,40 @@ async function createExamplePage(
     );
   }
 }
+
+/**
+ * Ask users if they want to set the tunnelRoute option.
+ * We can't set this by default because it increases hosting bills due to vercel charging for server-side rewrites.
+ * It's valuable enough to for users to justify asking the additional question.
+ */
+async function askShouldSetTunnelRoute() {
+  return await traceStep('ask-tunnelRoute-option', async () => {
+    const shouldSetTunnelRoute = await abortIfCancelled(
+      clack.select({
+        message:
+          'Do you want to route Sentry requests in the browser through your NextJS server to avoid ad blockers?',
+        options: [
+          {
+            label: 'Yes',
+            value: true,
+            hint: 'Can increase your hosting bill if you use Vercel',
+          },
+          {
+            label: 'No',
+            value: false,
+            hint: 'Browser errors and events might be blocked by ad blockers before being sent to Sentry',
+          },
+        ],
+        initialValue: true,
```

Comment:
```suggestion
        initialValue: false,
```

Reply:
Let's make it false please, I know this will cause more people to opt-out but it has to be explicit